plugin/hover.py

```diff
@@ -191,16 +190,14 @@ def hover_content(self) -> str:
                 value = item.get("value")
                 language = item.get("language")
 
-            if '\n' not in value:
-                value = "\n".join(textwrap.wrap(value, 80))
-
             if language:
                 formatted.append("```{}\n{}\n```\n".format(language, value))
             else:
                 formatted.append(value)
 
         if formatted:
-            return mdpopups.md2html(self.view, "\n".join(formatted))
+            frontmatter_config = "---\nallow_code_wrap: true\n---\n"
+            return mdpopups.md2html(self.view, frontmatter_config + "\n".join(formatted))
```

Comment:
I think it'd be just a tad bit nicer if this was using the `mdpopups.format_frontmatter` function.

Reply:
Done